include/zephyr/usb_c/usbc.h

```diff
@@ -220,6 +244,109 @@ typedef uint32_t (*policy_cb_get_rdo_t)(const struct device *dev);
  */
 typedef bool (*policy_cb_is_snk_at_default_t)(const struct device *dev);
 /** @}*/
+/** \addtogroup source_callbacks
```

Comment:
Add this doxygen group to the API reference of ucds.rst.

Also update the `dev` parameter documentation to `USB-C Connector Instance`

Reply:
Should the typedef be listed of the ucds.rst api, because they're in /** @cond INTERNAL_HIDDEN */ section of the include.

Reply:
These callbacks are part of the public API.  The should be moved out if the hidden section.

Reply:
done